wire-library/wire-grpc-server/src/main/java/com/squareup/wire/kotlin/grpcserver/WireMethodMarshaller.kt

```diff
@@ -0,0 +1,14 @@
+package com.squareup.wire.kotlin.grpcserver
+
+import io.grpc.MethodDescriptor
+
+/**
+ * Interface for generated method marshallers used in Google style grpc stubs.
+ *
+ * Adds a method for fetching the underlying class at run time to support
+ * generating other marshallers (like JSON encoding) from an instance of
+ * this class.
+ */
+interface WireMethodMarshaller<T>: MethodDescriptor.Marshaller<T> {
+  fun clazz(): Class<T>
```

Comment:
Feel free to ignore, but I find `clazz` to be not very useful in identifying what the class represents.  Perhaps something like `marshalledClass` or something like that.